Documentation/ServiceUtilities/MixedRealityServiceRegistryAndIMixedRealityServiceRegistrar.md

```diff
@@ -0,0 +1,54 @@
+# What are the MixedRealityServiceRegistry and IMixedRealityServiceRegistrar
+
+The Mixed Reality Toolkit has two very similarly named components that perform related tasks: 
+MixedRealityServiceRegistry and IMixedRealityServiceRegistrar. What are they and what purpose does each serve?
+
+## MixedRealityServiceRegistry
+
+The MixedRealityServiceRegistry is the component that contains instances of each registered service
+(core systems and extension services). It MixedRealiityServiceRegistry is implemented as a static C#
```

Comment:
either here or maybe else where call out that a service is defined by a IMixedRealityService?

Reply:
added